docs/se/reactivestreams/02_engine.adoc

```diff
@@ -0,0 +1,153 @@
+.Example of Multi usage:
+----
+AtomicInteger sum = new AtomicInteger();
+
+Multi.just("1", "2", "3", "4", "5")
+        .limit(3)
+        .map(Integer::parseInt)
+        .forEach(sum::addAndGet);
+
+System.out.println("Sum: " + sum.get());
+
+> Sum: 6
+----
+
+[source,java]
+.Example of Single usage:
+----
+Single.just("1")
+        .map(Integer::parseInt)
+        .map(i -> i + 5)
+        .toStage()
+        .whenComplete((i, t) -> System.out.println("Result: " + i));
+
+> Result: 6
+----
+
+[[terms]]
+.Operators
+|===
+|defer|Call the given supplier function for each individual downstream Subscriber to return a Flow.Publisher to subscribe to.
+|map|Map this `Multi` instance to a new `Multi` of another type using the given `Mapper`.
+|defaultIfEmpty|Signals the default item if the upstream is empty.
+|switchIfEmpty|Switch to the other publisher if the upstream is empty.
+|peek|Invoke provided consumer for every item in stream.
+|distinct|Filter out all duplicates.
+|filter|Filter stream items with provided predicate.
+|takeWhile|Take the longest prefix of elements from this stream that satisfy the given predicate. As long as predicate returns true, items from upstream are sent to downstream, when predicate returns false stream is completed.
+|dropWhile|Drop the longest prefix of elements from this stream that satisfy the given predicate. As long as predicate returns true, items from upstream are NOT sent to downstream but being dropped, predicate is never called again after it returns false for the first time.
+|limit|Limit stream to allow only specified number of items to pass.
+|skip|Skip first n items, all the others are emitted.
+|flatMap|Transform item with supplied function and flatten resulting `Flow.Publisher` to downstream.
+|flatMap|Transform item with supplied function and flatten resulting `Flow.Publisher` to downstream while limiting the maximum number of concurrent inner `Flow.Publisher`s and their in-flight item count, optionally aggregating and delaying all errors until all sources terminate.
+|flatMapIterable|Transform item with supplied function and flatten resulting `Iterable` to downstream.
+|observeOn|Re-emit the upstream's signals to the downstream on the given executor's thread using a default buffer size of 32 and errors skipping ahead of items.
+|observeOn|Re-emit the upstream's signals to the downstream on the given executor's thread.
+|flatMapIterable|Transform item with supplied function and flatten resulting `Iterable` to downstream.
+|forEach|Terminal stage, invokes provided consumer for every item in the stream.
+|collectList|Collect the items of this `Multi` instance into a `Single` of `List`.
+|collect|Collect the items of this `Multi` instance into a `Single`.
+|collect|Collect the items of this `Multi` into a collection provided via a `Supplier`
+*and mutated by a {@code BiConsumer} callback.
+|collectStream|Collects up upstream items with the help of a the callbacks of a `java.util.stream.Collector`.
```

Comment:
`help of a the callbacks` -> `help of the callbacks`

Reply:
thx